src/main/java/org/fogbowcloud/manager/core/models/Credential.java

```diff
@@ -1,5 +1,33 @@
 package org.fogbowcloud.manager.core.models;
 
 public class Credential {
+	private String name;
+	private boolean required;
+	private String valueDefault;
+	
+	public Credential(String name, boolean required, String valueDefault) {
+		this.name = name;
+		this.required = required;
+		this.valueDefault = valueDefault;
+	}
+	
+	public String getName() {
+		return name;
+	}
 
+	public void setName(String name) {
+		this.name = name;
+	}
+	
+	public String getValueDefault() {
+		return valueDefault;
+	}
+	
+	public void setValueDefault(String valueDefault) {
+		this.valueDefault = valueDefault;
+	}
+
+	public boolean isRequired() {
+		return required;
+	}
 }
```

Comment:
I did not check this code out to compile it. However, do we really need this set methods? Could not this class be immutable

Reply:
Done! Check [this](https://github.com/fogbow/fogbow-manager-core/commit/f091c029cb9afbf263381d4112277e16e340e449) out.